_data/master/navbars/getting-started.yml

```diff
@@ -33,6 +33,8 @@ toc:
       path: /getting-started/kubernetes/installation/vagrant/
   - title: Tutorials
     section:
+    - title: Using calicoctl
```

Comment:
I think this approach works well in the [Mesos docs](http://docs.projectcalico.org/master/getting-started/mesos#tutorials)